### ISSUE TITLE
Announcement

### ISSUE THREAD

**Issue by Chocobo1** [qbittorrent/wiki#1]
@sledgehammer999 @qbittorrent/lead-developers @qbittorrent/bug-handlers

This repo is now operative and it will become the source for the rendered wiki page at https://github.com/qbittorrent/qBittorrent/wiki
Originally, we left the wiki to be open for editing but a few malicious users kept disrupting and eventually we restrict editing to collaborators only. It was clear that the wiki changes should have someone looked at to avoid malicious intentions. Github doesn't provide enough options for maintaining the wiki and we have to set up a new repo for the review process.

I'm asking for team members to help maintain this repo. That is, reviewing, accepting and most importantly merging the PR submissions from users. You all are going to have write access to this PR (still in progress) and you'll be notified by Github after granting access.

Basically every change in this repo master branch will be synced to https://github.com/qbittorrent/qBittorrent/wiki. The sync will happen right away after merging a PR. And note that you should ***NOT*** use the edit button from the [rendered wiki](https://github.com/qbittorrent/qBittorrent/wiki) anymore. This repo will overwrite any changes from now on.

Currently I've set up basic CI that will check for typos and some minor issues, it should be enough help for reviewing PRs.

Lastly IMO the bar for PR acceptance shouldn't be too high, I would accept the changes if they are reasonable and without typos. The review process is not intended to become a burden for both PR author and the reviewer. (hopefully nobody will burnout from it)

**Comment by Chocobo1** [comment]
>You all are going to have write access to this PR (still in progress) and you'll be notified by Github after granting access.

It is done. Closing issue now.

**Comment by xero08** [comment]
I want to open a PR to address an obsolete dependency reference in one of the guides I'm following. How do I that ? I'm new here.

**Comment by Chocobo1** [comment]
>I want to open a PR to address an obsolete dependency reference in one of the guides I'm following. How do I that ? I'm new here.

https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository

**Comment by thalieht** [comment]
Are there any guidelines on when to use and when not to use each of the merge methods?

**Comment by xavier2k6** [comment]
> Are there any guidelines on when to use and when not to use each of the merge methods?

👍 , I may have made a faux pas in my "merging"

**Comment by Chocobo1** [comment]
>Are there any guidelines on when to use and when not to use each of the merge methods?

My personal preference (in general) is that:
* When there are multiple commits in a PR 
  * Each commit has its own individual value -> use Rebase Merge to preserve these commits.
  * Commits are just fixup of previous commit-> use Squash Merge to squash commits into one.
* Only one commit in a PR
  * If the commit title and body is well written -> use Rebase Merge.
  * If the commit title and body need serious improvement -> use Squash Merge to rewrite them.

Note that these rules only apply to this repo. Other repos may have other considerations.

**Comment by glassez** [comment]
In general, I agree with the above. There is one remark.
The following also requires that all the commit titles and bodies are well written:
> * Each commit has its own individual value -> use Rebase Merge to preserve these commits.
